### ISSUE TITLE
DN attribute missing

### ISSUE THREAD

**Issue by SirLefti** [txstate-etc/ldap-async#16]
In v1.x, returned objects also included the DN of the LDAP object. With v2.x, it returns objects of class `LdapEntry`, which do not provide this attribute, even when requesting it by attributes.

```js
let response = await ldapClient.search(`dc=domain,dc=com`, {
    filter: '(objectClass=person)',
    attributes: ['dn', 'mail', 'objectclass'],
    scope: 'sub',
    paged: false
})
console.log(response.every(node => node.get('dn') != undefined))
```

Am I missing something? Is there a different way to obtain the DN?

**Comment by wickning1** [comment]
That sounds like a bug. I’ll take a look.

**Comment by wickning1** [comment]
Fixed in 2.1.1
